File: atividade5/012.hs
apagar :: Int -> [t] -> [t]
apagar _ [] = []
apagar n (a:b)
    | n > 1     = apagar (n-1) b
    | otherwise = b

main = do
    print (apagar 3 [7, 9, 2, 4, 5, 3])

File: atividades/atividade16/012.pl
apagar(_, [], []) :- !.
apagar(0, L, L) :- !.
apagar(N, [_|T], TT) :- NN is N - 1, apagar(NN, T, TT).

% ?- apagar(0, [1,2,3,4,5], L).
% L = [1, 2, 3, 4, 5].

% ?- apagar(2, [1,2,3,4,5], L).
% L = [3, 4, 5].

% ?- apagar(0, [], L).
% L = [].

% ?- apagar(10, [1,2,3], L).
% L = [].
